### DISCUSSION TITLE
Pico2

### DISCUSSION THREAD

**Discussion by Flipje1955** [micropython/micropython#15713] [RP2040 / Pico]
Noting that the Pico2 Preview Build numbering does not follow that of Pico "1" nor that of other ports and the link to the github source gives "404 - page not found", does that mean MP is not yet "ready" for the Pico2 ? (Hope to get the Pico2 board this week).
thx !

**Comment by GM-Script-Writer-62850** [comment]
it look like the url changed 
```
https://www.raspberrypi.com/products/raspberry-pi-pico-2/ # 200 status
https://www.raspberrypi.com/products/raspberry-pi-pico2/ # 404 status
```
